directembedding/src/main/scala/ch/epfl/directembedding/Utils.scala

```diff
@@ -87,3 +93,16 @@ trait DirectEmbeddingUtils extends MacroModule with TransformationUtils {
   }
 
 }
+
+trait TypeHelper extends MacroModule {
+  import c.universe._
+
+  private def name(s: Symbol): String = s.name.toString.trim
+
+  def equalSymbols(a: Symbol)(b: Symbol): Boolean =
```

Comment:
Let's at least do a type of the symbol and if it is a method number of arguments. I think we could also add tests for different kind of weird situations. Also, I think it will be fun to see different kinds of symbols and how to compare them.